packages/lwc/index.js

```diff
@@ -44,34 +39,14 @@ function validateArgs(name, format, target, mode) {
             `Invalid target "${target}". Available targets: ${Array.from(TARGETS.keys())}`
         );
     }
-
-    if (!MODES.has(mode)) {
-        throw new Error(`Invalid mode "${mode}". Available modes: ${Array.from(MODES.keys())}`);
-    }
 }
 
-function getModule(name, format = DEFAULT_FORMAT, target = DEFAULT_TARGET, mode = DEFAULT_MODE) {
-    const distPath = getModulePath(name, format, target, mode);
-    return fs.readFileSync(distPath, 'utf8');
-}
-
-function getModulePath(
-    name,
-    format = DEFAULT_FORMAT,
-    target = DEFAULT_TARGET,
-    mode = DEFAULT_MODE
-) {
-    validateArgs(name, format, target, mode);
+function getModulePath(name, format = DEFAULT_FORMAT, target = DEFAULT_TARGET) {
```

Comment:
I think this would be a good effort for a breaking v3.0.0 release. Right now it's a bit scary because people can import whatever files they want that we happen to publish.

Opened an issue: https://github.com/salesforce/lwc/issues/2765